draft-ietf-quic-recovery.md

```diff
@@ -592,7 +592,7 @@ TCP's retransmission timeout period {{?RFC6298}}.
 
 ### Computing PTO
 
-When an ack-eliciting packet is transmitted, the sender schedules a timer for
```

Comment:
Can you undo the editorializing about sender/endpoint for this PR?  I prefer sender in this context, but I'd prefer to separate that question out.